static/src/javascripts/projects/commercial/modules/dfp/Advert.ts

```diff
@@ -228,7 +226,10 @@ class Advert {
 
 		this.sizes = sizeMapping;
 
-		this.slot.defineSizeMapping(buildGoogletagSizeMapping(sizeMapping));
+		const googleMapping = buildGoogletagSizeMapping(sizeMapping);
+		if (googleMapping) {
+			this.slot.defineSizeMapping(googleMapping);
```

Comment:
What do you think about this naming?

```suggestion
		const googletagSizeMapping = buildGoogletagSizeMapping(sizeMapping);
		if (googletagSizeMapping) {
			this.slot.defineSizeMapping(googletagSizeMapping);
```